sky/engine/core/painting/VertexMode.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of dart.sky;
+
+enum VertexMode {
```

Comment:
Please add /// dartdoc comments explaining what these are.  Maybe just link to the Skia documentation?